src/generic_ui/locales/en/messages.json

```diff
@@ -667,6 +667,14 @@
     "description": "Notification for when the user has been logged out of the social network.",
     "message": "You have been logged out of __network__"
   },
+  "PRE_LOG_OUT_WHEN_SHARING_WITH_ONE_OR_MANY": {
+    "description": "Notification to ask user if they are sure that they want to log out while they are sharing access with others.",
+    "message": "Are you sure you want to log out? You are sharing access with __name__ and __numOthers__ others"
+  },
+  "PRE_LOG_OUT_WHEN_NOT_SHARING_ACCESS": {
+    "description": "Notification to ask user if they are sure that they want to log out when sharing access with no-one.",
+    "message": "Are you sure you want to log out?"
+  },
```

Comment:
Oops, I'm so sorry for not being more clear!
We don't need a message for when the user is not sharing access, but we need one message for "SHARING_WITH_MANY" and another for "SHARING_WITH_ONE". I think changing PRE_LOG_OUT_WHEN_NOT_SHARING_ACCESS to SHARING_WITH_ONE and a message like "Are you sure you want to log out? You are sharing access with Bob." Otherwise, right now you would see "You are sharing with Bob and 0 others." which is a little funky looking. Let me know if that makes sense!